client/src/components/WowzaToggle.tsx

```diff
@@ -6,18 +6,19 @@ import { useHistory, useLocation } from "react-router-dom";
 /**
  * Determines whether a url corresponds to a new WOWZA portfolio mapping page vs an original.
  */
-export const isWowzaPath = (pathname: string) => removeLocalePrefix(pathname).startsWith("/wowza");
+export const isWowzaPath = (pathname: string) =>
```

Comment:
Let's change this to `isLegacyPath`, and then we can have a positive test here! Since we're not really using `wowza` in the path anymore, it's a tad confusing what this means